ci/test_utils.sh

```diff
@@ -20,5 +20,13 @@ print_system_stats() {
 
 print_ucx_config() {
   rapids-logger "UCX Version and Build Configuration"
-  ucx_info -v
+
+  set +e
+  which ucx_info > /dev/null
+  set -e
+  if [ $? -eq 0 ]; then
```

Comment:
Does it make sense to move these set commands to the wheel test script? Assuming that `set -e` is throwing off the `$?` check immediately below.

Alternatively, could probably go back to the check we were using in https://github.com/rapidsai/ucxx/pull/167/commits/5f8b0c9b42ac080c3296c164268c9c2fc6aa17c8, storing the value of `which ucx_info` in a variable:

```diff
diff --git a/ci/test_utils.sh b/ci/test_utils.sh
index df443a2..ac8c4f4 100755
--- a/ci/test_utils.sh
+++ b/ci/test_utils.sh
@@ -22,11 +22,11 @@ print_ucx_config() {
   rapids-logger "UCX Version and Build Configuration"
 
   set +e
-  which ucx_info > /dev/null
+  UCX_INFO_PATH=$(which ucx_info > /dev/null)
   set -e
-  if [ $? -eq 0 ]; then
-    ucx_info -v
-  else
+  if [ $UCX_INFO_PATH == "" ]; then
     echo "ucx_info not found"
+  else
+    ucx_info -v
   fi
 }
```

Reply:
> Does it make sense to move these set commands to the wheel test script? Assuming that `set -e` is throwing off the `$?` check immediately below.

I'm doing it like this now because I don't know exactly what's included in the wheels. The ideal scenario is that `ucx_info` would be available via the wheels install as well, so I'm not yet sure whether we're missing UCX entirely or if it's just `ucx_info` and want to see first if we can get tests running. I'm guessing @vyasr will be able to tell us what we can or can't do with the wheels sometime later today.

> Alternatively, could probably go back to the check we were using in [5f8b0c9](https://github.com/rapidsai/ucxx/commit/5f8b0c9b42ac080c3296c164268c9c2fc6aa17c8), storing the value of `which ucx_info` in a variable:

There's actually a much simpler solution that I overlooked initially, see https://github.com/rapidsai/ucxx/pull/167/commits/1da02e6014066c958cdd9822648d15d8dfe4b8d8 .